sdg/schemas/SchemaInputBase.py

```diff
@@ -170,3 +171,27 @@ def get_value_translation(self, field):
             raise ValueError(field + " field does not have options element")
 
         return {x['enum'][0]: x['translation_key'] for x in options}
+
+
+    def add_field_order(self, field):
```

Comment:
Consider renaming this function to something that makes it clear that you're adding something to the field_order list. `add_item_to_field_order` or similar?

Reply:
Thanks, just renamed it. As for why, just a general habit of trying to use methods instead of directly working with properties. I don't always remember to do that, but I try. No particular reason other than that.